source/code/source/playfab/PlayFabEventPipeline.cpp

```diff
@@ -110,6 +110,30 @@ namespace PlayFab
             auto playFabEmitEventResponse = std::make_shared<PlayFabEmitEventResponse>();
             playFabEmitEventResponse->emitEventResult = emitResult;
 
+            std::shared_ptr<PlayFabError> emitEventError  = std::make_shared<PlayFabError>();
+
+            emitEventError->ErrorName = "PlayFabEventPipeline IntakeEvent Error";
+            emitEventError->ErrorMessage = "PlayFabEventPipeline did not accept the event. Please see ErrorDetails for more information.";
+            emitEventError->ErrorCode = PlayFabErrorCode::PlayFabErrorEventNotFound;
+
+            emitEventError->HttpCode = 0;
+            emitEventError->HttpStatus = "None";
+
+            if(emitResult == EmitEventResult::Overflow)
+            {
+                emitEventError->ErrorDetails = "PlayFabEventPipeline was unable to take the event due to memory limits. Please wait for batching to complete before retrying or increase the pipeline size.";
+            }
+            else if (emitResult == EmitEventResult::Disabled)
```

Comment:
>EmitEventResult::Disabled [](start = 35, length = 25)

is this result even possible anymore? I went looking through the source code for PlayFabEventBuffer and it doesn't look like there's actually any code which flips the `disabled` flag necessary to trigger this result

Reply:
I noticed that as well. There's no way to really toggle that disabled flag on the Pipeline at the moment.